opendocdb-cts/Makefile

```diff
@@ -19,11 +21,11 @@ run-fmt:
 	go run . --dir=../cts convert ../cts-mongosh
 
 run-mongodb:
-	go run . --dir=../cts run --uri=mongodb://127.0.0.1:27001/qklck6njgh9o --golden
-	go run . --dir=../cts run --uri=mongodb://127.0.0.1:27001/cts
+	go run . --dir=../cts run --uri=mongodb://${HOST}:27001/qklck6njgh9o --golden
```

Comment:
This line is curious. It uses `go run` which expects `go` to be installed in the host. Then uses dev container which installs `golang` extension.

Let me see if I understand the scenario, we want to allow tests to run in the dev container without needing to use `go` from host? Possibly I'm misunderstanding.

Reply:
> Let me see if I understand the scenario, we want to allow tests to run in the dev container without needing to use `go` from host?

That's pretty much what DevContainers are for – to be able to do development inside container, including the ability to run tests